test/access-control.integration.js

```diff
@@ -111,9 +111,15 @@ describe('access control - integration', function() {
           assert.equal(user.password, undefined);
         });
       });
+
+      // user has replaceOnPUT = false; so then both PUT and PATCH should be allowed for update
       lt.describe.whenCalledRemotely('PUT', '/api/users/:id', function() {
         lt.it.shouldBeAllowed();
       });
+
+      lt.describe.whenCalledRemotely('PATCH', '/api/users/:id', function() {
+        lt.it.shouldBeAllowed();
+      });
```

Comment:
> I think we should test the endpoints for `replaceOnPUT:false` too.
I already have tests for `replaceOnPUT:false` Please see:
https://github.com/strongloop/loopback/pull/2316/files#diff-eeafd17907dda5058f6e881724ba8c6eR245